lib/tweetsclient/mysql_track.py

```diff
@@ -7,14 +7,8 @@
 Copyright (c) 2010 __MyCompanyName__. All rights reserved.
 """
 
-import sys
-import os
-import unittest
-import ConfigParser
 import MySQLdb
 
-import anyjson
-import beanstalkc
```

Comment:
What's the story with these?

Reply:
I removed these since they aren't being used anywhere. This shouldn't be in this PR so I'll remove them.